pkg/agent/agent.go

```diff
@@ -875,48 +891,47 @@ func (i *Initializer) initNodeLocalConfig() error {
 		return err
 	}
 	i.nodeConfig.NodeMTU = mtu
-	klog.Infof("Setting Node MTU=%d", mtu)
+	klog.InfoS("Setting Node MTU", "MTU", mtu)
 
 	if i.networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
 		return nil
 	}
 
-	// Parse all PodCIDRs first, so that we could support IPv4/IPv6 dual-stack configurations.
+	// Parse all PodCIDRs first, so that we can support IPv4/IPv6 dual-stack configurations.
 	if node.Spec.PodCIDRs != nil {
 		for _, podCIDR := range node.Spec.PodCIDRs {
 			_, localSubnet, err := net.ParseCIDR(podCIDR)
 			if err != nil {
-				klog.Errorf("Failed to parse subnet from CIDR string %s: %v", node.Spec.PodCIDR, err)
+				klog.ErrorS(err, "Failed to parse subnet from Pod CIDR string", "CIDR", podCIDR)
 				return err
 			}
 			if localSubnet.IP.To4() != nil {
 				if i.nodeConfig.PodIPv4CIDR != nil {
-					klog.Warningf("One IPv4 PodCIDR is already configured on this Node, ignore the IPv4 Subnet CIDR %s", localSubnet.String())
+					klog.InfoS("One IPv4 PodCIDR is already configured on this Node, ignore the IPv4 Subnet CIDR", "subnet", localSubnet)
 				} else {
 					i.nodeConfig.PodIPv4CIDR = localSubnet
-					klog.V(2).Infof("Configure IPv4 Subnet CIDR %s on this Node", localSubnet.String())
+					klog.V(2).InfoS("Configured IPv4 Subnet CIDR on this Node", "subnet", localSubnet)
 				}
 				continue
 			}
 			if i.nodeConfig.PodIPv6CIDR != nil {
-				klog.Warningf("One IPv6 PodCIDR is already configured on this Node, ignore the IPv6 subnet CIDR %s", localSubnet.String())
+				klog.InfoS("One IPv6 PodCIDR is already configured on this Node, ignore the IPv4 Subnet CIDR", "subnet", localSubnet)
```

Comment:
```suggestion
				klog.InfoS("One IPv6 PodCIDR is already configured on this Node, ignore the IPv6 Subnet CIDR", "subnet", localSubnet)
```
otherwise it would mean dual-stack is not supported.

Reply:
thanks, copy-paste error